profiles/base.yaml

```diff
@@ -1448,3 +1448,37 @@ tarantula_noun: biomechanical tarantula
 tarantula_startup_delay: 15
 tarantula_no_use_scripts:
 tarantula_debug: false
+
+# Number of minutes to wait between performing a verb on an item.
+faux_atmo_interval: 15
+
+# Items that don't have an inherent atmo effect
+# but whose verbs if done periodically would.
+# faux_atmo_items:
```

Comment:
💡 I would keep the setting itself uncommented

Reply:
To confirm, the example config would look this then? I wasn't sure if people would know to remove the `[]` if they were to uncomment the below lines to specify their items/verbs.
```yaml
# Items that don't have an inherent atmo effect
# but whose verbs if done periodically would.
faux_atmo_items: []
#   - name: blue basilisk
#     verbs:
#     - pet
#     - pinch
#     - poke
#   - name: hooded cloak
#     verbs:
#     - rub
#     - pat
#     - wave
```